packages/send/frontend/src/stores/auth-store.ts

```diff
@@ -202,6 +229,41 @@ export const useAuthStore = defineStore('auth', () => {
     }
   }
 
+  /**
+   * Save authenticated user data obtained from token bridge
+   */
+  async function storeUser(rawAuthData: RawAuthData) {
```

Comment:
This is the function called in `background.ts` that does the actual instantiation and storing of the authenticated `User`.